helm-charts/splunk-otel-collector/values.yaml

```
@@ -371,7 +371,9 @@ logsCollection:
     #     containerName:
     #       value: server
     #     firstEntryRegex: ^[^\s].*
-
+    # Set `splunk.com/include` annotation to true on pod and `logsCollection.containers.useSplunkIncludeAnnotation` flag to `true` to include its logs from ingested to your Splunk platform deployment.
```

It's not only about Splunk Platform. How about this wording?
```suggestion
    # Set this flag to `true` to collect logs from pods with `splunk.com/include: true` annotation and ignore others.
```